content/html/concepts/elements/terms/input/input.md

```diff
@@ -16,15 +16,15 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-The **`<input>`** element creates an interactive element, usually used within a [form](https://www.codecademy.com/resources/docs/html/forms) to allow user input. It can be used to make text boxes, [color pickers](https://www.codecademy.com/resources/docs/uiux/color-picker), date pickers, and other [UI](https://www.codecademy.com/resources/docs/uiux/ui-design) elements.
+The **`<input>`** element creates an interactive element, usually used within a [form](https://www.codecademy.com/resources/docs/html/forms) to allow user input.It can be used to make text boxes, [color pickers](https://www.codecademy.com/resources/docs/uiux/color-picker), date pickers, and other [UI](https://www.codecademy.com/resources/docs/uiux/ui-design) elements. Also, the `for` attributes are linking the [<label> elements](https://www.codecademy.com/resources/docs/html/elements/label) to the `id`of the <input> elements.
```

Comment:
```suggestion
The **`<input>`** element creates an interactive element, usually used within a [form](https://www.codecademy.com/resources/docs/html/forms) to allow user input.It can be used to make text boxes, [color pickers](https://www.codecademy.com/resources/docs/uiux/color-picker), date pickers, and other [UI](https://www.codecademy.com/resources/docs/uiux/ui-design) elements.
```
@CBID2 you still need to remove this addition to the opening paragraph.